storage_bucket/create.py

```diff
@@ -23,21 +18,20 @@ def __call__(
         storage_bucket_name: str,
         location: str,
         storage_class: str = 'STANDARD',
-    ) -> ResultE[Bucket]:
+    ) -> Bucket:
         """List the storage bucket files."""
-        return flow(
-            self.get_client(),
-            bind(partial(
-                self._create_bucket,
-                name=storage_bucket_name,
-                storage_class=storage_class,
-                location=location,
-            )),
+        client = self.get_client()
+
+        return self._create_bucket(
+            client=client,
+            name=storage_bucket_name,
+            storage_class=storage_class,
+            location=location,
         )
 
-    @safe
     def _create_bucket(
         self,
+        *,
```

Comment:
What does this syntax do?

Reply:
It bans the use of positional arguments, so one can only use keyword arguments. 
It makes it safer to use the code since a change in order of arguments won't break anyone's code